x/group/keeper/keeper.go

```diff
@@ -204,6 +206,12 @@ func NewKeeper(storeKey storetypes.StoreKey, cdc codec.Codec, router *authmiddle
 	}
 	k.voteTable = *voteTable
 
+	if config.MaxMetadataLen != 0 {
+		k.config = config
+	} else {
+		k.config = DefaultConfig()
+	}
```

Comment:
let's just set MaxMetadataLen in config instead of setting the full struct with `DefaultConfig()`
see how it's done in https://github.com/cosmos/cosmos-sdk/pull/11093/files